core/imt/inc/ROOT/TThreadExecutor.hxx

```diff
@@ -121,7 +121,7 @@ namespace ROOT {
       auto Map(F func, const std::vector<T> &args, R redfunc, unsigned nChunks) -> std::vector<typename std::result_of<F(T)>::type>;
 
       // Functions that interface with the parallel library used as a backend
-      void   ParallelFor(unsigned start, unsigned end, unsigned step, const std::function<void(unsigned int i)> &f);
+      void   ParallelFor(unsigned int start, unsigned int end, unsigned step, const std::function<void(unsigned int i)> &f);
```

Comment:
I don't understand this change; can we get `doxygen` to handle plain `unchanged` instead?

Reply:
I think the problem is that the signature in the .cxx is different than in the .hxx:

https://github.com/root-project/root/blob/6dd09ce44f0e35df43e0703ad3621f10c60f6290/core/imt/src/TThreadExecutor.cxx#L161

vs

https://github.com/root-project/root/blob/3e1b2b5061729cb9241dfec779a3481ae694684c/core/imt/inc/ROOT/TThreadExecutor.hxx#L124

So Doxygen does not match those well. Should we change the .cxx instead of the .hxx ? Or is this mismatch done on purpose, and I should revert the change ?

Reply:
Thanks for explaining, @ferdymercury ! ROOT doesn't really seem to have a preference - I personally prefer `unsigned` over `unsigned int`, so could you normalize to that, please?

Reply:
Sure, fixed now.